app/controllers/posts_controller.rb

```diff
@@ -10,6 +10,12 @@ def index
   # GET /posts/1
   # GET /posts/1.json
   def show
```

Comment:
Hey,
Are you trying to display all of the comments relevant to a post?
I see a potential area of improvement here.
What happens if the user visits a post that is not found in the database?
As it appears currently, the code could fail through.
I think it would be beneficial to wrap this block into a try-catch clause.
This area is usually prone to ActiveRecordNotFound errors.
You can do the following:
```rb
begin
  <code block>
rescue Exception => e
  <code to rescue>
end
```
If this is not clear, I would be more than happy to point you to the most common errors that occur here.
Cheers,
Alex